interface.py

```diff
@@ -11,6 +11,7 @@ def __init__(self):
         self.my_game_round = None
         self.winner = None
         self.timer = False
+        self.flag = 0
```

Comment:
```suggestion
        self.flag_count = 0
```
Samotne "flag" nedava moc najevo, jestli jde o sebranou vlajku, nebo nejakou jinou, navic ma v sobe robot atribut "flags". Asi bych to udelala, jak mame v backendu, aby bylo jasne, ze jde o pocet vlajek k dosazeni.